soweego/linker/feature_extraction.py

```diff
@@ -327,3 +328,123 @@ def intersection_percentage_size(pair):
             return count_intersect / count_total if count_total > 0 else np.nan
 
         return fillna(concatenated.apply(intersection_percentage_size), self.missing_value)
+
+
+class OccupationCompare(BaseCompareFeature):
+
+    name = "OccupationCompare"
```

Comment:
Please stick to `recordlinkage` naming "conventions": convert the new class name into snake case, e.g., `OccupationCompare` -> `occupation_compare`

Reply:
thanks! I've made this change for the Occupations and DateCompare features, in 0b6263fd7e44be4adb5d294d88c60c9413aadc4b